### ISSUE TITLE
'Import using Script' not importing files

### ISSUE THREAD

**Issue by the-raging-bull** [Sonarr/Sonarr#7418]
### Is there an existing issue for this?

- [X] I have searched the existing open and closed issues

### Current Behavior

I remux all my content from MKV to MP4 (I'm sure people will have comments, but that's what meets my needs) then complete some maintenance tasks on things like extracted subtitles

All that works well, what I am now wanting to do is trigger this on the `Import using script` functionality. I've scoured the internet and cannot seem to find any example scripts, and best i can come to the below is a bug, but entirely possible I am just doing it wrong

I have reviewed the code at 
https://github.com/Sonarr/Sonarr/blob/develop/src/NzbDrone.Core/MediaFiles/ScriptImportDecider.cs#L186-L218 and from what i can tell from https://github.com/Sonarr/Sonarr/blob/develop/src/NzbDrone.Core/MediaFiles/ScriptImportDecider.cs#L50 I need the output of the script to contain some particular strings to do the next part.

I am writing the below as output of the script post conversion, but this is throwing an error on import that it is not a child of the server folder
```
26/11/2024 4:15:27 AM - Standard - [MediaFile]C:\tmp\SeriesName.S03E01.720p.HEVC.x265-MeGusta\SeriesName.S03E01.720p.HEVC.x265-MeGusta.mp4
26/11/2024 4:15:27 AM - Standard - [ExtraFile]C:\tmp\SeriesName.S03E01.720p.HEVC.x265-MeGusta\SeriesName.S03E01.720p.HEVC.x265-MeGusta.eng.srt
```
**Weirdly** (and what leads me to think this is a bug), the ExtraFile(s) specified will always be imported - just not the mp4

I've also tried moving the files using the script itself (not really what i want to do) and then running with `[MoveStatus]RenameRequested` but a rename doesn't occur. None of the options are commented or similar so its not super clear what each of the options in https://github.com/Sonarr/Sonarr/blob/develop/src/NzbDrone.Core/MediaFiles/ScriptImportDecision.cs do

I tried changing the `sonarr_sourcepath` and `sonarr_destinationpath` environment variables too (thinking maybe they were passed back) but that didnt work

Ideally i want to automate the conversion from mkv to mp4+srt and then allow sonarr to import directly from the folder. 

Note in the example here `c:\tmp` is the download directory and the transcode directory - i just replace the mkv with an mp4 

All the below aside, i cant even get an import to work with the simple script in "Steps to reproduce"

### Expected Behavior

The custom import script should be able to return a converted `MediaFile` and Sonarr imports the file

### Steps To Reproduce

custom powershell example
```powershell
$ErrorActionPreference = "SilentlyContinue"

if ($env:sonarr_eventtype -eq "test") { exit 0 }

Write-Output "[MediaFile]$($env:sonarr_sourcepath)"
exit 0
```

### Environment

```markdown
- OS: Windows 10
- Sonarr: 4.0.10.2544
- Docker Install: No
- Using Reverse Proxy:  No
- Browser: NA
- Database: Sqlite 3.36.0
```

### What branch are you running?

Main

### Trace Logs?

First Error thows  `Script output contains non-existent media file` (the file is definitely there)
```
24-11-26 15:15:16.2|Warn|ImportApprovedEpisodes|Couldn't import episode C:\tmp\SeriesName.S03E01.720p.HEVC.x265-MeGusta\SeriesName.S03E01.720p.HEVC.x265-MeGusta.mkv

[v4.0.10.2544] NzbDrone.Core.MediaFiles.ScriptImportException: Script output contains non-existent media file: C:\tmp\SeriesName.S03E01.720p.HEVC.x265-MeGusta\SeriesName.S03E01.720p.HEVC.x265-MeGusta.mkv
   at NzbDrone.Core.MediaFiles.ImportScriptService.ProcessOutput(List`1 processOutputLines) in ./Sonarr.Core/MediaFiles/ScriptImportDecider.cs:line 78
   at NzbDrone.Core.MediaFiles.ImportScriptService.TryImport(String sourcePath, String destinationFilePath, LocalEpisode localEpisode, EpisodeFile episodeFile, TransferMode mode) in ./Sonarr.Core/MediaFiles/ScriptImportDecider.cs:line 0
   at NzbDrone.Core.MediaFiles.EpisodeFileMovingService.TransferFile(EpisodeFile episodeFile, Series series, List`1 episodes, String destinationFilePath, TransferMode mode, LocalEpisode localEpisode) in ./Sonarr.Core/MediaFiles/EpisodeFileMovingService.cs:line 174
   at NzbDrone.Core.MediaFiles.EpisodeFileMovingService.MoveEpisodeFile(EpisodeFile episodeFile, LocalEpisode localEpisode) in ./Sonarr.Core/MediaFiles/EpisodeFileMovingService.cs:line 85
   at NzbDrone.Core.MediaFiles.UpgradeMediaFileService.UpgradeEpisodeFile(EpisodeFile episodeFile, LocalEpisode localEpisode, Boolean copyOnly) in ./Sonarr.Core/MediaFiles/UpgradeMediaFileService.cs:line 80
   at NzbDrone.Core.MediaFiles.EpisodeImport.ImportApprovedEpisodes.Import(List`1 decisions, Boolean newDownload, DownloadClientItem downloadClientItem, ImportMode importMode) in ./Sonarr.Core/MediaFiles/EpisodeImport/ImportApprovedEpisodes.cs:line 159
```

Then Second Error appears to accept that i changed the path and throws that the `is not a child of`
```
26/11/2024 4:15:27 AM - Standard - [MediaFile]C:\tmp\SeriesName.S03E01.720p.HEVC.x265-MeGusta\SeriesName.S03E01.720p.HEVC.x265-MeGusta.mp4
2024-11-26 15:15:27.1|Debug|AggregateLanguages|Language couldn't be parsed from release, fallback to series original language: English
2024-11-26 15:15:27.1|Warn|ImportApprovedEpisodes|Couldn't import episode C:\tmp\SeriesName.S03E01.S03E01.720p.HEVC.x265-MeGusta\SeriesName.S03E01.S03E01.720p.HEVC.x265-MeGusta.mp4

[v4.0.10.2544] NzbDrone.Common.Disk.NotParentException: C:\tmp\SeriesName.S03E01.720p.HEVC.x265-MeGusta\SeriesName.S03E01.S03E01.720p.HEVC.x265-MeGusta.mp4 is not a child of \\SERVER01\TV\Series Name
   at NzbDrone.Common.Extensions.PathExtensions.GetRelativePath(String parentPath, String childPath) in ./Sonarr.Common/Extensions/PathExtensions.cs:line 85
   at NzbDrone.Core.MediaFiles.ImportScriptService.TryImport(String sourcePath, String destinationFilePath, LocalEpisode localEpisode, EpisodeFile episodeFile, TransferMode mode) in ./Sonarr.Core/MediaFiles/ScriptImportDecider.cs:line 0
   at NzbDrone.Core.MediaFiles.EpisodeFileMovingService.TransferFile(EpisodeFile episodeFile, Series series, List`1 episodes, String destinationFilePath, TransferMode mode, LocalEpisode localEpisode) in ./Sonarr.Core/MediaFiles/EpisodeFileMovingService.cs:line 174
   at NzbDrone.Core.MediaFiles.EpisodeFileMovingService.MoveEpisodeFile(EpisodeFile episodeFile, LocalEpisode localEpisode) in ./Sonarr.Core/MediaFiles/EpisodeFileMovingService.cs:line 85
   at NzbDrone.Core.MediaFiles.UpgradeMediaFileService.UpgradeEpisodeFile(EpisodeFile episodeFile, LocalEpisode localEpisode, Boolean copyOnly) in ./Sonarr.Core/MediaFiles/UpgradeMediaFileService.cs:line 80
   at NzbDrone.Core.MediaFiles.EpisodeImport.ImportApprovedEpisodes.Import(List`1 decisions, Boolean newDownload, DownloadClientItem downloadClientItem, ImportMode importMode) in ./Sonarr.Core/MediaFiles/EpisodeImport/ImportApprovedEpisodes.cs:line 159
```

### Anything else?

Permissions should not be a factor as the Sonarr service is running under the context of my windows user and not the default system context

**Comment by markus101** [comment]
@JeWe37 is the most knowledgeable as the primary author/user of it and may be able to give some insight.

**Comment by the-raging-bull** [comment]
thanks @markus101 can defintily see why it may look that way - i did a fair bit of troubleshooting which led me to believe this is a bug - from what i can see in the source code returning `[MediaFile]` should result in that file being sorted

@JeWe37 do you mind taking a look at the description here to confirm if this is a `bug` report or `support` request
